src/TypeMapping/PostgresTypeMapper.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace staabm\PHPStanDba\TypeMapping;
+
+use PHPStan\ShouldNotHappenException;
+use PHPStan\Type\Type;
+
+final class PostgresTypeMapper implements TypeMapper
+{
+    public function mapToPHPStanType(int $type, int $flags, int $length): Type
+    {
+        throw new ShouldNotHappenException();
```

Comment:
I don't know whether PDO supports this `$flags` and `$type` constants like `mysqli` does... does PDO support this very same concept?

Reply:
I believe we could get this info with this method https://www.php.net/manual/en/pdostatement.getcolumnmeta.php , but not sure if it's going to work atm.

Reply:
We habe this metadata already available in the mysqli type mapper - I guess

Reply:
Turned differently: in case pdo uses the same flag values, we might even share the type-mapper between the 2 reflector impls